packages/components/src/components/checklist/checklist.jsx

```diff
@@ -17,7 +17,11 @@ const ItemStatus = ({ status, onClick, button_text }) => {
         case 'action':
         default:
             return (
-                <div className='dc-checklist__item-status--action' onClick={onClick}>
+                <div
+                    className='dc-checklist__item-status--action'
+                    onClick={onClick}
+                    data-testid='dc-checklist__item-status--action'
```

Comment:
@hamid-deriv could you pls change data-testid to `data-testid='dt_checklist_item_status_action'` because i already use it in my Withdrawal UI TS migration card which is in Ready stage now.

Reply:
Fixed.